packages/pn-commons/src/utility/paymentCaching.utility.ts

```diff
@@ -0,0 +1,133 @@
+import _ from 'lodash';
```

Comment:
Check if all function exported are used outside of this file. For example isTimestampWithin2Minutes is not used outside